crm_timesheet/views/crm_lead_view.xml

```diff
@@ -0,0 +1,177 @@
+                    class="oe_stat_button"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                />
+            </div>
+            <!-- New fields -->
+            <field name="user_id" position="after">
+                <field name="project_id" invisible="1" />
+                <field
+                    name="project_id"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                />
+            </field>
+            <page name="extra" position="after">
+                <page string="Timesheet" groups="hr_timesheet.group_hr_timesheet_user">
+                    <field name="timesheet_ids" invisible="1" />
+                    <field
+                        name="timesheet_ids"
+                        groups="hr_timesheet.group_hr_timesheet_user"
+                        context="{'default_project_id': project_id, 'default_user_id': uid, 'tree_view_ref': 'hr_timesheet.timesheet_view_tree_user', 'form_view_ref': 'hr_timesheet.hr_timesheet_line_form'}"
+                    />
+                </page>
+            </page>
+        </field>
+    </record>
+    <!-- Leads -->
+    <record id="view_crm_lead_kanban" model="ir.ui.view">
+        <field name="name">Add timesheet time controls to lead kanban</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.view_crm_lead_kanban" />
+        <field name="arch" type="xml">
+            <xpath expr="//footer" position="inside">
+                <field name="show_time_control" invisible="1" />
+                <a
+                    name="button_start_work"
+                    invisible="show_time_control != 'start'"
+                    class="o_kanban_inline_block text-success"
+                    string="Start work"
+                    tabindex="-1"
+                    type="object"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <i class="fa fa-lg fa-play-circle" title="Start" />
+                </a>
+                <a
+                    name="button_end_work"
+                    invisible="show_time_control != 'stop'"
+                    class="o_kanban_inline_block text-warning"
+                    string="Stop work"
+                    tabindex="-1"
+                    type="object"
+                    groups="hr_timesheet.group_hr_timesheet_user"
+                >
+                    <i class="fa fa-lg fa-stop-circle" title="Stop" />
+                </a>
+            </xpath>
+        </field>
+    </record>
+    <record id="crm_case_tree_view_leads" model="ir.ui.view">
+        <field name="name">Add timesheet time controls to lead tree</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_tree_view_leads" />
+        <field name="arch" type="xml">
+            <list position="inside">
+                <field name="show_time_control" invisible="1" />
```

Comment:
@jdebetaz like for hr_timesheet_view.xml, we must change `invisible` to `column_invisible`

```suggestion
                <field name="show_time_control" column_invisible="1" />
```